<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="AboutWidget" name="widgetAbout" native="true"/>
    </item>
    <item>
     <widget class="HistoryWidget" name="widgetHistory" native="true"/>
    </item>
    <item>
     <widget class="WebWidget" name="widgetWeb" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuw00t">
    <property name="title">
     <string>w00t</string>
    </property>
    <widget class="QMenu" name="menuNavigation">
     <property name="title">
      <string>Navigation</string>
     </property>
     <addaction name="actionToggle_navigation"/>
     <addaction name="actionStop"/>
    </widget>
    <addaction name="actionBack"/>
    <addaction name="actionForward"/>
    <addaction name="actionSettings"/>
    <addaction name="actionNew_window"/>
    <addaction name="actionReload_page"/>
    <addaction name="actionExit"/>
    <addaction name="menuNavigation"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuw00t"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReload_page">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionNew_window">
   <property name="text">
    <string>New window</string>
   </property>
  </action>
  <action name="actionToggle_navigation">
   <property name="text">
    <string>Toggle navigation</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WebWidget</class>
   <extends>QWidget</extends>
   <header>webwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HistoryWidget</class>
   <extends>QWidget</extends>
   <header>historywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AboutWidget</class>
   <extends>QWidget</extends>
   <header>aboutwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
